### ISSUE TITLE
Feature request: invisible flag removes flatride bases but still draws ride

### ISSUE THREAD

**Issue by spacek531** [OpenRCT2/OpenRCT2#13858]
Many New Element Designs users like to replace the vanilla bases with custom bases that look like concrete. There is a good likelihood this is a more desired feature than hiding parts of a ride, as corrupt elements do currently.

When the new save file format and the invisibility flag are implemented, this seems like it should be the default behavior of the invisibility flag as applied to flat ride track pieces.

**Comment by duncanspumpkin** [comment]
I believe this has already been implemented (yesterday?) by @Gymnasiast for the new save format.

**Comment by spacek531** [comment]
Looks like it's still a little buggy but it's cool to see it's happening.

**Comment by spacek531** [comment]
I'm told this is also relevant, probably a nicer way to do it than this https://github.com/OpenRCT2/OpenRCT2/issues/8407

**Comment by Gymnasiast** [comment]
Indeed. I have now implemented it for the flat rides (the actual ride stations are to follow).

It’s only a matter of selecting the appropriate station style, no TI needed:

![image](https://user-images.githubusercontent.com/1478678/127173315-17ad1a6a-ad32-4ddc-9aa9-7177c2d00d74.png)
